.github/workflows/close-stale.yml

```
@@ -5,10 +5,12 @@ on:
     - cron: "8 7 * * *" # arbitrary time not to DDOS GitHub
 
 permissions:
-  issues: write
-  pull-requests: write
+  contents: read
 jobs:
   stale:
+    permissions:
+      issues: write # required for closing stale issues
+      pull-requests: write # required for closing stale pull requests
```

(just trying to preserve existing formatting as much as possible)
```suggestion
      issues: write
      pull-requests: write
```